### PR TITLE
#22 Add global styling, home page, and global button

### PR DESCRIPTION
**Pull request by franc-hoon**
- Created the background
- Created a home page with the link "/home"
- Created the global button

### CODE REVIEW

**Review by thakurtushar02 — APPROVED** [review]
lgtm

File: src/stylesheets/app.scss [hunk 1]
```diff
@@ -1,32 +1,77 @@
...
   box-sizing: border-box;
 }
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: "Hind Madurai", sans-serif;
+  font-size: 36px;
```
Comment (max-gurr) [inline src/stylesheets/app.scss]: I think this should be changed to 16px, as this is the standard font-size for websites. We can scale up element font-sizes by using relative sizes, e.g. 2em.
If we set `font-size: 36px` here then it'll carry through to every other element, which will make it hard to put in regular sized text

File: src/stylesheets/app.scss [hunk 2]
```diff
@@ -1,32 +1,77 @@
...
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
 
-.app {
+.background {
   text-align: center;
```
Comment (max-gurr) [inline src/stylesheets/app.scss]: Can you move this into `.main-content`?

File: src/stylesheets/app.scss [hunk 3]
```diff
@@ -1,32 +1,77 @@
...
   height: 100vh;
-  overflow: hidden;
+}
+
+.circle1,
+.circle2 {
+  height: 40rem;
+  width: 40rem;
+  position: absolute;
+  border-radius: 9999px;
```
Comment (max-gurr) [inline src/stylesheets/app.scss]: why 9999px instead of 50%?

File: src/stylesheets/variables.scss [hunk 4]
```diff
@@ -1,3 +1,5 @@
-$background-colour: #2E2D40;
-$background-gradient: radial-gradient(circle at 10% 20%, #373563 0%, transparent 50%),
-                      radial-gradient(circle at 90% 80%, #04022e 0%, $background-colour 40%);
\ No newline at end of file
+$background: linear-gradient(225deg, #2e2d40 0%, #3c3a61 100%);
+$background-circle1: linear-gradient(280deg, #3f3e5c 0%, #3a3c4a 100%);
```
Comment (max-gurr) [inline src/stylesheets/variables.scss]: I can't see where `$background-circle1` and `$background-circle2` have been used?

**Review by max-gurr — CHANGES_REQUESTED** [review]

**Review by max-gurr — APPROVED** [review]

**Comment by max-gurr** [comment]
Thanks for addressing those comments, good job!

**Pull request merged**